text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+
+### Confusion about emberjs/rfcs and ember-cli/rfcs
+
+The Ember project currently has two separate RFC processes for Ember.js and Ember CLI.
+
+This leads to confusion due to having to keep track of two different repositories to track progress.
+It also increases the burden on the submitter of an RFC, as the two processes have diverged slightly,
+making the submitter be aware of these differences during the lifecycle of the RFC.
+
+There are also some cases where it is not clear exactly which repository the RFC should be submitted to.
+This can also lead to multiple RFCs being written, one for each repository.
+While multiple RFCs might be warranted in some cases,
+it should be a choice instead of a limitation of the current processes.
+
+### Process does not cover sufficient areas of development
+
+RFCs to emberjs/rfcs and ember-cli/rfcs concern themselves mostly with features or deprecations to ember.js and ember-cli respectfully, with some ember-data proposals in emberjs/rfcs.
+
+Once again taking inspiration from Rust, they have experimented expanding the types of RFCs proposed
+[crates.io default ranking](https://github.com/rust-lang/rfcs/blob/master/text/1824-crates.io-default-ranking.md)
+[default URLs for documentation](https://github.com/rust-lang/rfcs/blob/master/text/1826-change-doc-default-urls.md).
+The latter one, for example, would have been a useful RFC when designing the new [API docs](https://github.com/ember-learn/ember-api-docs).
+
+Rust has RFCs for documentation, website, internals, tools.
+We only have emberjs and ember-cli due to them having been implemented separately.
+
+This makes it so that a potential contributor does not know where to submit a proposal.
+
+### Lingering RFCs
+
+The emberjs/rfcs repository currently has [54 issues](https://github.com/emberjs/rfcs/issues) and [62 PRs](https://github.com/emberjs/rfcs/pulls) open, and ember-cli/rfcs has [183 issues](https://github.com/ember-cli/ember-cli/issues) and [49 PRs](https://github.com/ember-cli/ember-cli/pulls).
+A significant percentage of these have not been active in the recent past.
+
+We have kept PRs and issues open so people could more easily find the discussions,
+but this has instead given a negative feeling of staleness, as RFCs linger open without new feedback.
+
+## Detailed design
+
+To address the issues raised in the “Motivation” section, I will propose changes to the current process.
+
+### Implement Core Champion
+
+To make sure that RFCs receive adequate support from the team, Ember CLI has implemented the idea of a champion associated with each RFC.
+The goal is that in seeking a champion from the team,
+the RFC author starts a dialogue with the team and gets some early feedback.
+The champion is then responsible for representing the RFC in team meetings, and for shepherding its progress.
```

Comment:
no comma

Reply:
Also as a user, I would want to see formal affirmation that a decent number of core team members _would_ champion other RFCs before I'd feel good about this section. If this falls to just a few people, it will not work and we'll burn out them out

Reply:
It's kind of a side comment. I'm hoping that team members will chime in and say, "yes I want to do this," not necessarily that people should be listed here